packages/web-components/src/slider/slider.spec.md

```diff
@@ -0,0 +1,118 @@
+# Slider
+
+## Component Description
+
+The Slider component allows users to quickly select a value by dragging an icon across a bar. It is often used when setting values with a relaxed precision such as audio volume and screen brightness.
+
+## Design Spec
+
+[Link to Design Spec in Figma](https://www.figma.com/file/IBCBJxEbPKS7CvLHG55Wn9/Slider?node-id=513%3A510&t=hCOhOiQJVCVNVVJP-0)
+
+## Engineering Spec
+
+### Inputs
+
+- @attr size: "small" "medium" | "medium"
+- @attr vertical: boolean | false
+- @attr current-value: number
+- @attr min: number
+- @attr max: number
+- @attr step: number | 1
+  - If set, slider appearance has ticks for each step value
+- @attr disabled: boolean | false
+- @attr mode: "single-value" "range" | "single-value"
+
+### Outputs
+
+None
+
+### Events
+
+- @change: Fires whenever the slider's value is changed
+
+### Slots
+
+- default: the primary slot intended for the `input` element
+- thumb: the draggable element of the slider
+- lower-thumb: same as `thumb`, but used in `range` mode
+- upper-thumb: same as `thumb`, but used in `range` mode
+- track: the slider's base and used to display the min/max selectable values
+- progress-track: the progress indicator when in `range` mode
+
+### CSS Variables
+
+None
+
+### Interactions
+
+#### thumb
+
+- On :hover a tooltip appears indicating the current value of the slider
+
+#### track
+
+- Clicking on the track moves the thumb to the location of mouse click
+
+## Accessibility
+
+- [x] Find the matching component through WCAG's patterns: https://www.w3.org/WAI/ARIA/apg/patterns/
+  - https://www.w3.org/WAI/ARIA/apg/patterns/slider/
+- [x] Are there any accessibility elements unique to this component?
+- [x] List ARIA attributes
+  - [x] role="slider" in FAST
+  - [x] aria-valuenow in FAST
+  - [x] aria-valuemin in FAST
+  - [x] aria-valuemax in FAST
+  - [x] aria-orientation (if vertical) in FAST
+- [ ] Does the component support 400% zoom?
+- [ ] What keyboard behaviors does the component support?
+  - [ ] Up / Right : Increments slider by Step amount
+  - [ ] Down / Left: Decrements slider by Step amount
+  - [ ] PageUp/Up/Right & Shift : Increments the value of the slider by 10 \_ step.
+  - [ ] PageDown /Down/Left & Shift : Decrements the value of the slider by 10 \_ step.
+  - [ ] Home : Sets value to the min prop.
+  - [ ] End : Sets value to the max prop.
+
+## Preparation
+
+- [x] [Find the base FAST Component](https://explore.fast.design/components/) this component will inherit from and document
+  - [FAST Slider Component](https://explore.fast.design/components/fast-slider)
+- [x] [Check the Fluent UI React V9 Component Spec](https://github.com/microsoft/fluentui/tree/master/specs) for differences and document
+  - [Fluent Slider Spec](https://github.com/microsoft/fluentui/blob/master/packages/react-components/react-slider/docs/Spec.md)
+  - no `defaultValue` attribute, instead only `current-value` that specifies the selected range of the slider
```

Comment:
`value` is what sets the default value. I believe that defaultValue is a react construct for inputs. There's a long issue  here which illustrates the entire evolution of why we are keeping `value` as initial/default value instead of changing how it works: https://github.com/microsoft/fast/issues/5119

Reply:
TLDR, if we change how `value` works in an HTML custom element, it's unlikely that the platform will change to align with us - doing so would "break the internet". In this case, we elected to add a new attribute which has a better chance of making it into the platform over time than changing how value works.

Reply:
I need more clarity here. If we go ahead with the `current-value` attribute from FAST, is it just a matter of reflecting those deviations from Fluent React in the Spec?

Reply:
I guess my thought here is - we need to decide if this is an area worth forking in order to maintain compatability with react. `defaultValue` in my understanding here is a _react_ construct for inputs - the custom elements leverage Form Association and so there is no construct of "defaultValue". On the HTML platform, `value` is literally the default/initial value...it doesn't two way bind, it doesn't update, etc. It is an "input" to the input which gives a default value from the view for a given form element. That's just how the platform works. IMO this is an earned difference and we can document as such. FAST in this case is aligned to how the web platform works today and how it will work moving forward (I don't see a world where an input's value attribute changes in behavior, again this will "break the internet"). This is a _react_ deviation from the platform to enable certain features. I'm not sure how we want to treat this, but from a standards perspective, the deviation here isn't around how value works.

Reply:
I see the arguments for sticking with the web platform, but I don't see the arguments for maintaining compatibility with React, especially if it's going against the grain. @miroslavstastny do you have any strong opinions on maintaining compatibility with React on this, or are you OK with documenting the differences?

Reply:
The `defaultValue` is React-specific concept to support a default value for an [uncontrolled component](https://reactjs.org/docs/uncontrolled-components.html).
If you pass `value` prop to a React component, it becomes controlled - the component itself cannot change the `value`, it is controlled from the parent. The `defaultValue` allows as to define a default/initial value while keeping the component as uncontrolled.

As this is a React specific concept, we should not follow the pattern in web components and do whatever is natural there.